snap/snapcraft.yaml

```diff
@@ -54,7 +52,7 @@ parts:
       # Old versions of this file used to unstage
       # lib/python3.8/site-packages/augeas.py to avoid conflicts between
       # python-augeas 0.5.0 which was pinned in snap-constraints.txt and
-      # Robie's python-augeas fork which creates an auto-generated cffi file at
+      # our python-augeas fork which creates an auto-generated cffi file at
```

Comment:
Actually, since we're now based on a newer version of `python-augeas`, are we sure we don't need to put the line back? I don't quite remember the conditions where having version not having it caused things to break; we should probably double check that, if you happen to remember how to do that.

Reply:
Yes I am sure that we don't need to put the line back.

The problem in the past was that we were installing two versions of python-augeas in [different](https://github.com/basak/certbot-snap-build/blob/4c52664161f52bd5b4da33c0edefbfb3e74c986d/snap/snapcraft.yaml#L40) [parts](https://github.com/basak/certbot-snap-build/blob/4c52664161f52bd5b4da33c0edefbfb3e74c986d/snap/snapcraft.yaml#L65) and then trying to combine them in the final snap which we're no longer doing with everything being in a single part.